box/boxnavigator.py

```diff
@@ -43,12 +43,15 @@ def __init__(self, position: Pt, rotation: float, env: BoxEnv) -> None:
 
         # TODO: find appropriate values for these
         self.distance_threshold = 75
```

Comment:
Should this be an argument?

Reply:
This is the threshold in how far one is allowed to be from the target before it allows the next target to be activated. This can definitely be made into an argument.